x-pack/plugins/case/server/routes/api/utils.ts

```diff
@@ -101,7 +101,7 @@ export const transformCases = (
 });
 
 export const flattenCaseSavedObjects = (
-  savedObjects: SavedObjectsFindResponse<CaseAttributes>['saved_objects'],
+  savedObjects: Array<SavedObject<CaseAttributes>>,
```

Comment:
Not sure why `SavedObjectsFindResponse<CaseAttributes>['saved_objects']` was used previously, but adapted back to `SavedObject` which is the expected type.